Products/ZenModel/ZenossInfo.py

```diff
@@ -279,6 +281,7 @@ def getZopeVersion(self):
         major, minor, micro, status, release = version.getZopeVersion()
         return Version(name, major, minor, micro)
 
+    @FunctionCache('getZenossRevision')
```

Comment:
It will never work on a customer site, even with git. I'm not sure it is worth the effort to print out the revision.